cross/armel/toolchain.cmake

```diff
@@ -16,7 +16,7 @@ add_compile_options(--sysroot=${CROSS_ROOTFS})
 set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} -target ${TOOLCHAIN}")
 set(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} --sysroot=${CROSS_ROOTFS}")
 
-if("${CLI_CMAKE_RUNTIME_ID}" MATCHES "tizen.*")
```

Comment:
@gkhanna79 I don't think using gcc-arm-linux-gnueabi is a right way, because it may mislead toolchain to use of installed gcc binaries (objects and libaries) instead of binaries in rootfs and it may cause runtime. Also options to clang, i.e. `-B` and `-L`, always have wrong value for Tizen.

In CoreCLR and CoreFX, they already use environmental variable in cmake files.
https://github.com/dotnet/coreclr/blob/master/cross/armel/toolchain.cmake#L25
https://github.com/dotnet/corefx/blob/master/cross/armel/toolchain.cmake#L18

Therefore I think using environmental variable here suggested by @jyoungyun is reasonable if there is no side-effect to other build process.

In addition, we can reduce size of Docker image for CI if we don't need gcc-arm-linux-gnueabi.